File: .net & mono/Weed3Demo/demo_plus/table/user_info.cs
using Noear.Weed;
using Weed3Demo.demo;

namespace Weed3Demo.demo_ex {
    //此类，可由工具根据数据表生成
    //
    public class user_info : DbTable {
        public user_info() : base(DbConfig.test) {
            table("$.user_info u");
            set("userID", () => userID);
            set("sex", () => sex); 
        }

        public long? userID;
        public int? sex;
    }
}

File: java/Weed3Demo/src/weed3demo/demo_plus/table/user_info.java
package weed3demo.demo_plus.table;

import noear.weed.DbTable;
import weed3demo.config.DbConfig;

/**
 * Created by yuety on 2017/7/22.
 */
public class user_info extends DbTable {
    public user_info() {
        super(DbConfig.test);

        table("$.user_info u");
        set("userID", () -> userID);
        set("sex", () -> sex);
    }

    public Integer userID;
    public Integer sex;
}
